internal_packages/message-list/lib/message-timestamp.cjsx

```diff
@@ -15,30 +15,9 @@ class MessageTimestamp extends React.Component
     +nextProps.date isnt +@props.date or nextProps.isDetailed isnt @props.isDetailed
 
   render: =>
-    msgDate = moment.tz(@props.date, Utils.timeZone)
-    nowDate = @_today()
-    formattedDate = @_formattedDate(msgDate, nowDate, @props.isDetailed)
     <div className={@props.className}
```

Comment:
It looks like we lost the isDetailed option here—we may need to check and see where / if that was in use anywhere.